howto.md

```diff
@@ -55,7 +55,16 @@ Every proposal repository is supposed to be a fork of the main `spec` repo. Unfo
       git merge upstream/master
       ```
 
-2. Create Overview:
+2. If your proposal has dependencies on another active proposal, whose repository is named `<<parent-proposal>>`, then add that as an additional remote repository:
+
+   1. Run:
+      ```
+      git remote add <<parent-proposal> https://github.com/WebAssembly/<<parent-proposal>>.git
```

Comment:
nit: missing `>` in `<<parent-proposal>>`

Reply:
Done.